google/cloud/sql/connector/InstanceConnectionManager.py

```diff
@@ -122,27 +124,48 @@ def _get_metadata(
                 + "proj_name (str) and inst_name (str)."
             )
 
-        req = service.instances().get(project=project, instance=instance)
-        res = req.execute()
+        if not credentials.valid:
+            request = google.auth.transport.requests.Request()
+            credentials.refresh(request)
+
+        bearer_token = credentials.token
+
+        headers = {
+            "Authorization": "Bearer %s" % bearer_token,
+            "Content-Type": "application/json",
+        }
+
+        url = "https://www.googleapis.com/sql/v1beta4/projects/%s/instances/%s" % (
```

Comment:
Can we use `format` instead `%`?

Reply:
Based on the launch of this, if we only do 3.6+ this could start using f-strings: https://www.python.org/dev/peps/pep-0498/

Also, setup.py likely should specify which versions this supports. My guess though is the plan is 3.5+ since 3.5 won't EOL for a bit yet.